integration-testing/red1/project.clj

```diff
@@ -0,0 +1,4 @@
+(defproject red1 "n/a"
+  :description "I should fail due to outdated :dependencies"
+  :dependencies [[org.clojure/clojure "1.10.2"]]
```

Comment:
`1.10.3` is the latest for now
```suggestion
  :dependencies [[org.clojure/clojure "1.10.3"]]
```

Reply:
All "red" projects are intentionally outdated, the script verifies that antq does in fact point out so :)

Reply:
Oops, I misread that!